renderer/containers/preferences.js

```diff
@@ -244,7 +245,14 @@ export default class PreferencesContainer extends Container {
   updateShortcut = async (setting, shortcut) => {
     try {
       await ipc.callMain('update-shortcut', {setting, shortcut});
-      this.setState({[setting]: shortcut});
+      this.setState({
```

Comment:
Just use the callback version of `setState`.

Reply:
We are on Unstated v1, which doesn't support the callback style. While doing some work on refactoring the cropper, editor and exports, I'm trying to see if we can start a general refactoring, including switching to unstated-next and hooks whenever possible, but that still needs work

Reply:
👍